backend/src/API/coffeeChatAPI.ts

```diff
@@ -145,61 +179,135 @@ export const checkMemberMeetsCategory = async (
-    );
-  }
-  if (category === 'cuapts member') {
-    return otherMember?.subteams.includes('cuapts');
+
+  // If otherMember and submitter don't exist, status should stay undefined
+  if (otherMember && submitter) {
+    if (category === 'an alumni') {
+      status = (await allMembers()).every((member) => member.email !== otherMember.email)
+        ? 'pass'
+        : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not an alumni`);
+    } else if (category === 'courseplan member') {
+      status = otherMember.subteams.includes('courseplan') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember?.lastName} is not a CoursePlan member`);
+    } else if (category === 'business member') {
+      status = otherMember.role === 'business' ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a business member`);
+    } else if (category === 'idol member') {
+      status = otherMember.subteams.includes('idol') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not an IDOL member`);
+    } else if (category === 'curaise member') {
+      status = otherMember.subteams.includes('curaise') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a CURaise member`);
+    } else if (category === 'cornellgo member') {
+      status = otherMember.subteams.includes('cornellgo') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a CornellGo member`);
+    } else if (category === 'carriage member') {
+      status = otherMember.subteams.includes('carriage') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a Carriage member`);
+    } else if (category === 'qmi member') {
+      status = otherMember.subteams.includes('queuemein') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a QMI member`);
+    } else if (category === 'cuapts member') {
+      status = otherMember.subteams.includes('cuapts') ? 'pass' : 'fail';
+      status === 'fail' &&
+        (message = `${otherMember.firstName} ${otherMember.lastName} is not a CUApts member`);
+    } else if (category === 'a pm (not your team)') {
+      const isPm = otherMember.role === 'pm';
+      const notSameTeam = haveNoCommonSubteams(submitter, otherMember);
+      status = isPm && notSameTeam ? 'pass' : 'fail';
+      if (status === 'fail') {
+        if (!isPm && !notSameTeam) {
+          message = `${otherMember.firstName} ${otherMember.lastName} is not a PM and is on the same team as ${submitter?.firstName} ${submitter?.lastName}`;
```

Comment:
I feel like we don't need to get that specific here. For a category like this, the person not being a PM is already enough to justify why the coffee chat category is wrong. I don't think it's necessary to state both reasons